packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -237,6 +238,14 @@ export type TableProps<
    * @beta
    */
   enableVirtualization?: boolean;
+  /**
+   * Initialize the table scrolled to this item.
+   *
```

Comment:
No need empty lines

Reply:
The empty lines provide a new line when shown in intellisense
![image](https://user-images.githubusercontent.com/15036346/172888326-a1905552-3e30-452d-9c6e-550de62a3eef.png)